src/styles/global.css

```diff
@@ -100,6 +104,11 @@
       dark:text-dark-on-surface dark:bg-dark-surface;
   }
 
+  .container-surface-variant {
```

Comment:
Here could be 
```css
.container-surface-variant, .card-tonal {
    ...
}
```

Reply:
I can't do that because `card-tonal` is a component, not a utility.